modules/claims_api/app/sidekiq/claims_api/claim_uploader.rb

```diff
@@ -7,7 +7,7 @@ module ClaimsApi
   class ClaimUploader < ClaimsApi::ServiceBase
     sidekiq_options retry: true, unique_until: :success
 
-    def perform(uuid)
+    def perform(uuid, original_filename)
```

Comment:
do we have to pass this in? can't we grab it from the file_data or file_name? particularly looking at something [like this](https://github.com/department-of-veterans-affairs/vets-api/blob/5a0835cbd5ebdb8fd48c922ccde5d8a12a09e4fe/modules/claims_api/app/sidekiq/claims_api/claim_uploader.rb#L24)? 

also curious, I believe this is generated by a method, right? we're not taking user input here are we? - if so, we have concerns with plaintext arguments being passed to the Sidekiq UI which could include PII.

Reply:
We overwrite the filename prior to upload to s3, so for example a 526 attachment PUT yields the following by the time we get to BD (in this case the original attachment name from Postman was "kitchen_sink.pdf":
(byebug) claim.file_data['filename']
"019847ea2aa1d7b63289101416924e55.pdf"

The new filename is created via a new method in BD called `generate_file_name`, so attachments are of the form `"#{veteran_name}_#{claim_id}_#{original_filename}.pdf"`. The original_filename bit comes from the name of the uploaded attachment. Not sure how else to implement the requirement noted in the dev notes:
Attachment uploads should retain the original file name from the request to Claims API as part of the uploaded file name